glide.yaml

```diff
@@ -52,9 +52,9 @@ import:
 - package: github.com/emicklei/go-restful
   version: v1.2
 - package: k8s.io/client-go
-  version: 82aa063804cf055e16e8911250f888bc216e8b61
+  version: v7.0.0
```

Comment:
I think Tom was asking us to comment any pins from now on, which I think would be a good idea.  Did you figure out why the machinery pin was needed?